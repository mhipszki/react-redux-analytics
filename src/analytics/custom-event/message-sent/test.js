import { transform } from './';

test('constructs `message_sent` payload to be pushed to data layer', () => {
  const state = {
    message: {
      text: 'message text'
    }
  };
  const action = { timestamp: 1234567890 };
  expect(transform(state, action)).toMatchSnapshot();
});
