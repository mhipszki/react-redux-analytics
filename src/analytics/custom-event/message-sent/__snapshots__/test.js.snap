// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constructs \`message_sent\` payload to be pushed to data layer 1`] = `
Object {
  "event": "message_sent",
  "message": Object {
    "text": "message text",
    "timestamp": "Thu Jan 15 1970",
  },
}
`;
